src/Integration.Vsix/CFamily/FileConfig.cs

```diff
@@ -85,7 +85,7 @@ public static FileConfig TryGet(ProjectItem projectItem, string absoluteFilePath
                 };
             }
 
-            private static MethodInfo getEvaluatedPropertyValue;
+            internal static MethodInfo getEvaluatedPropertyValue;
```

Comment:
I think it's worth including the "for testing" comment i.e. `internal /* for testing */ MethodInfo ...`
It captures the design intent that the property is expected to be called from test code but not from other product code. Without the comment, it's not clear whether the choice of scope was clearly thought about or not (and in some bits of the code base it doesn't seem to have been).

Reply:
... or alternatively change the scope back to private and add an internal `SetEvaluatePropertyValueDelegateForTesting` method.

Reply:
Better still, you never actually set the value from the tests, so you only need an internal method to clear the private static property.